CHANGES.rst

```diff
@@ -1,6 +1,7 @@
 1.5.3 (unreleased)
 ==================
 
+
```

Comment:
Huh. Are 2 blank lines required below a release heading?

Reply:
Hmm, that might have been the result of a poorly-attempted merge conflict... 😬

Reply:
No biggy. Can be fixed in the next PR.